targets/rp2350.ld

```diff
@@ -0,0 +1,45 @@
+/* See Rust for a more complete reference: https://github.com/rp-rs/rp-hal/blob/main/rp235x-hal-examples/memory.x */
+MEMORY
+{
+   /* 2MiB safe default.  */
```

Comment:
Are these known to match the machine hardware? This is one of the places you really want to get it right, or you'll be in Debug Hell forever.

Reply:
Yes. 

`the RP2354 devices have internal 2MB flash memories` via https://datasheets.raspberrypi.com/rp2350/hardware-design-with-rp2350.pdf page 10